test/int/wizards/variables/variablesVerificator.ts

```diff
@@ -0,0 +1,79 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as _ from 'lodash';
+import { expect } from 'chai';
+import { DebugProtocol } from 'vscode-debugprotocol';
+import { trimWhitespaceAndComments } from '../breakpoints/implementation/printedTestInputl';
+import { ManyVariablesValues, } from './variablesWizard';
+import { printVariables } from './variablesPrinting';
+
+/** Whether the expected variables should match exactly the actual variables of the debuggee
+ * or whether the expected variables should only be a subset of the actual variables of the debuggee
+ */
+export enum KindOfVerification {
+    SameAndExact, /** Same exact variables */
+    ProperSubset /** Expected variables are a subset of the actual variables */
+}
+
+/**
+ * Provide methods to validate that the variables appearing on the stack trace are what we expect
+ */
+export class VariablesVerificator {
+    /** Verify that the actual variables are exactly the variables that we expect */
+    public assertVariablesAre(variables: DebugProtocol.Variable[], expectedVariablesPrinted: string | ManyVariablesValues): void {
```

Comment:
parameter should be called "expectedVariables" since we don't know if it is printed or not yet

Reply:
Fixed